### DISCUSSION TITLE
ActionButton upgrade

### DISCUSSION THREAD

**Discussion by qqzii** [moonshine-software/moonshine#865] [Ideas]
```
Column::make([
    Text::make('label1', 'value1'),
    Text::make('label2', 'value2'),
    ActionButton::make('handle')->method('updateSomething')
])
```

it would be nice to be able to access fields values ​​and keys in updateSomething()

**Comment by lee-to** [comment]
Я оставлю это обсуждение и пока подумаю но в целом такое решается через собственный обработчик
->onClick('youFunction') и уже в js вы собираете данные с нужных вам полей и отправляете запросом на обработчик

**Reply by lee-to** [reply]
Хотя одна идея есть, возможно в ближайшем релизе что-нибудь придумаем

**Comment by lee-to** [comment]
https://github.com/moonshine-software/moonshine/pull/869
